### DISCUSSION TITLE
Regenerate incorrect pcode generated by old version of ghidra

### DISCUSSION THREAD

**Discussion by 12Me21** [NationalSecurityAgency/ghidra#7148] [Q&A]
Is there a way to regenerate all pcode in a project, without having to clear the instructions?

This project was started in a version of ghidra which didn't handle 68000 byte moves with autoincrement correctly. (among other things) 
e.g. here it should decrement A0 by 1, but instead decremented it by 2:
<pre>
    00006260 11 00               move.b        D0b,-(A0)
                                                      $U24d00:1 = COPY D0b
                                                      A0 = INT_SUB A0, 2:4 <i>← should be 1:4</i>
                                                      $U8b80:1 = COPY $U24d00:1
                                                      STORE ram(A0), $U8b80:1
                                                      NF = INT_SLESS $U24d00:1, 0:1
                                                      ZF = INT_EQUAL $U24d00:1, 0:1
                                                      VF = COPY 0:1
                                                      CF = COPY 0:1
</pre>
That bug has since been fixed, but the pcode in this project is still incorrect (and this affects the decompiler as well)

If i clear the instruction and disassemble it again, the pcode is correct:
<pre>
    00006260 11 00               move.b        D0b,-(A0)
                                                      $U24d00:1 = COPY D0b
                                                      A0 = INT_SUB A0, <b>1</b>:4
                                                      $U8c80:1 = COPY $U24d00:1
                                                      STORE ram(A0), $U8c80:1
                                                      NF = INT_SLESS $U24d00:1, 0:1
                                                      ZF = INT_EQUAL $U24d00:1, 0:1
                                                      VF = COPY 0:1
                                                      CF = COPY 0:1
</pre>
But i can't just do that to the entire project, because that also deletes references etc.

**Comment by f-raZ0R** [comment]
Bumping this discussion because I'm encountering this myself with 11.3 fixing armv6's vmul/vsqrt, i have to re-disassemble every vmul instruction to fix the registers it's taking
